pkg/instance/kubernetes.go

```diff
@@ -214,9 +225,6 @@ func (ks kubernetesService) pause(instance *model.Instance) error {
 		}
 	}
 
-	labelSelector = fmt.Sprintf("app.kubernetes.io/instance=%s-database", instance.Name)
-	listOptions = metav1.ListOptions{LabelSelector: labelSelector}
-
 	sets := ks.client.AppsV1().StatefulSets(instance.GroupName)
 	setsList, err := sets.List(context.TODO(), listOptions)
```

Comment:
Could we, and should we, adopt the same naming convention as above to get rid of the list suffix?

Reply:
I don't think this works here as we need the `sets` var to pass it to the scale func
It works above as we just chain the list operation directly on the `Pods()` operation

Reply:
:+1: